atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -33,6 +34,12 @@
 class GroupManifestsPlugin(PostBuildPlugin):
     is_allowed_to_fail = False
     key = PLUGIN_GROUP_MANIFESTS_KEY
+    manifest_media_types = [
+        MEDIA_TYPE_DOCKER_V2_SCHEMA2,
+        MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST,
+        MEDIA_TYPE_OCI_V1,
+        MEDIA_TYPE_OCI_V1_INDEX
```

Comment:
Probably want to leave MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST and MEDIA_TYPE_OCI_V1_INDEX out of this list.

Reply:
Agree, workers wouldn't produce manifest lists / OCI indexes anyway. Implemented

Reply:
Had to revert this - these media types are also used to fetch the digest of the created manifest list / OCI index, so it has to include media types for those - otherwise tests fail